packages/terra-table/tests/wdio/table-spec.js

```diff
@@ -0,0 +1,94 @@
+Terra.describeViewports('Table', ['medium', 'large'], () => {
+  describe('Default Table', () => {
+    it('Validates the default table', () => {
+      browser.url('/raw/tests/cerner-terra-framework-docs/table/default-table');
+      Terra.validates.element('default-table', '#default-terra-table');
+    });
+  });
+
+  describe('No Interaction Table', () => {
+    it('Validates the default table is not interactable', () => {
+      browser.url('/raw/tests/cerner-terra-framework-docs/table/no-interaction-table');
+      browser.keys(['Tab']);
+      expect($('#focused-button')).toBeFocused();
```

Comment:
This will not validate the functionality.  You need something like

expect($('#focused-button').toBeFocused()).toBe(true);